server/server.go

```diff
@@ -498,6 +498,7 @@ func (s *Server) Run() error {
 		return err
 	}
 
+	s.initConfig()
```

Comment:
Will this overwrite the updated configuration in etcd throught `pd-ctl`?

Reply:
yep, I think I should close this pr, since  https://github.com/pingcap/tidb/issues/29523 is not a bug